src/optimize/bundles_route/bundles_route.js

```diff
@@ -85,6 +85,12 @@ export function createBundlesRoute({
         fileHashCache
       )
     ),
+    buildRouteForBundles(
+      `${basePublicPath}/bundles/core/`,
+      `/bundles/core/`,
+      resolve(__dirname, join('..', '..', 'core', 'public', 'target', 'public')),
```

Comment:
optional: let's use `fromRoot` to avoid fragile relative paths https://github.com/elastic/kibana/blob/4e8ab56497cb7bc4d0fb99a72ef69f8de36c0d59/src/optimize/index.js#L48